packages/hash/api/src/model/knowledge/user.model.ts

```diff
@@ -129,12 +130,18 @@ export default class extends EntityModel {
    *
    * @param params.emails - the emails of the user
    * @param params.kratosIdentityId - the kratos identity id of the user
+   * @param params.isInstanceAdmin (optional) - whether or not the user is an instance admin of the workspace (defaults to `false`)
    */
   static async createUser(
     graphApi: GraphApi,
     params: UserModelCreateParams,
   ): Promise<UserModel> {
-    const { emails, kratosIdentityId, actorId } = params;
+    const {
+      emails,
+      kratosIdentityId,
+      actorId,
+      isInstanceAdmin = false,
```

Comment:
I'm curious, I haven't seen this kind of destructuring with default values often. Can it be done at arbitrary depth?  As in `{top: {something: {defaultThisValue = true}}}`? :)

Reply:
I think so, yes :)